mne/io/pick.py

```diff
@@ -60,7 +60,11 @@ def channel_type(info, idx):
     elif kind == FIFF.FIFFV_SYST_CH:
         return 'syst'
     elif kind in [FIFF.FIFFV_SEEG_CH, 702]:  # 702 was used before v0.11
```

Comment:
Ahh we might have, yeah. In that case I can just remove the 702 from this line and it should work right?